test/sample-data.json

```diff
@@ -242,6 +242,94 @@
             }
           }
         }
+      },
+      "featureWithImports": {
+        "__compat": {
+          "description": "This sub-feature has imports",
+          "support": {
+            "webview_android": {
```

Comment:
This should be last, not first.

Reply:
Yeah, this was written work on&nbsp;https://github.com/mdn/browser-compat-data/pull/1882 began.